lib/cocoapods/sandbox/file_accessor.rb

```diff
@@ -1,3 +1,5 @@
+autoload :MachO, 'macho'
```

Comment:
this should probably just be required in `dynamic_binary?`, as this is what we do for things like nap too.